pkg/registry/authorization/util/helpers.go

```diff
@@ -77,3 +77,13 @@ func AuthorizationAttributesFrom(spec authorizationapi.SubjectAccessReviewSpec)
 
 	return authorizationAttributes
 }
+
+const AllVersion = "*"
+
+// MatchAllVersionIfEmpty returns a "*" if the version is unspecified
+func MatchAllVersionIfEmpty(version string) string {
+	if len(version) == 0 {
+		return AllVersion
+	}
+	return version
+}
```

Comment:
I don't think we need an exported method for this, I would suggest doing this locally in the method where it is used